### PR TITLE
[TC-1703] extension user must have oper role

### PR DESCRIPTION
**Pull request by zhilhuan**
fix for #1703

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/825/
Test PASSed.

File: traffic_ops/app/db/seeds.sql [hunk 1]
```diff
@@ -402,7 +402,7 @@ insert into type (name, description, use_in_table) values ('STEERING_WEIGHT', 'W
 insert into type (name, description, use_in_table) values ('STEERING_ORDER', 'Ordered steering target', 'steering_target') ON CONFLICT (name) DO NOTHING;
 
 -- users
-insert into tm_user (username, role, full_name, token) values ('extension', 3, 'Extension User, DO NOT DELETE', '91504CE6-8E4A-46B2-9F9F-FE7C15228498') ON CONFLICT DO NOTHING;
+insert into tm_user (username, role, full_name, token) values ('extension', 2, 'Extension User, DO NOT DELETE', '91504CE6-8E4A-46B2-9F9F-FE7C15228498') ON CONFLICT DO NOTHING;
```
Comment (mitchell852) [inline traffic_ops/app/db/seeds.sql]: hardcoding an ID does't seem like a good idea and in my opinion it should have never been done this way in the first place. maybe you can do something like:

insert into tm_user (username, role, full_name, token) values ('extension', (select id from role where name = 'operations'),

**Comment by zhilhuan** [comment]
Thanks for the comments! 
Updated.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/827/
Test PASSed.

**Pull request merged**